packages/terra-consumer-nav/src/components/nav-logo/NavLogo.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import styles from './NavLogo.scss';
+
+const propTypes = {
+  /**
+   *  The path to where the image is located.
+   */
+  path: PropTypes.string,
+  /**
+   *  The alternate text that is read by screen readers.
+   */
+  altText: PropTypes.string,
+  /**
+   * Whether or not the logo should be placed on top of a white card.
+   */
+  isCard: PropTypes.bool,
+};
+
+const defaultProps = {
+  path: 'http://placeholder.pics/svg/220x140/FF0606-FFFFFF',
+  altText: 'Client Logo',
+  isCard: false,
+};
+
+const cx = classNames.bind(styles);
+
+const NavLogo = ({
+  path,
+  altText,
+  isCard,
+  ...customProps
+}) => (
+  <div className={cx('logo-container', isCard ? 'logo-card' : '')} {...customProps}>
```

Comment:
make the card a separate component... if its a card, return the logo nested in the card, else just return the logo.

Eventually we will hvae a card component and will use that one and delete this one